node/configuration.yaml

```diff
@@ -14817,7 +14817,7 @@ mainnet_wallet_macos64: &mainnet_wallet_macos64
   update:
     <<:
     applicationName: csl-daedalus
-    applicationVersion: 3
+    applicationVersion: 5
```

Comment:
Why did you change it here and on line 14808? It should be changed only for staging (i. e. `mainnet_dryrun…` configurations), no?

Reply:
@gromakovsky, good catch!  Thank you!